test/test_hyphenation_and_stress.py

```diff
@@ -28,35 +28,43 @@ def ids_func(val):
 
 
 def spanish_common_words():
-    with data_file_open("spanish-hyphens.txt") as fin:
+    with data_file_open("spanish-hyphens2.txt") as fin:
         for line in fin:
-            word, hyphenation, stressed = line.strip().split()
-            yield word, hyphenation, int(stressed)
+            if line.startswith("#"):
+                continue
+            word, hyphenation, stressed, accent_pos = line.strip().split()
+            try:
+                accent_pos = int(accent_pos)
+            except:
```

Comment:
Since this is controlled input, we could do:
```
accent_post = int(accent_pos) if accent pos != '-' else None
```
If you prefer the exception way, please specify `ValueError` in the `except` line.

I leave the choice up to you :)